### PR TITLE
chore(ci): add workflow to tag codeowners for docs changes

### LINKED ISSUE #1121: feat: CODEOWNERS for translation
I would like to set up code owners for the translations. Being a code owner means you get a notification from GitHub when there is a PR with changes to the files of your language, so you can review them. If you want to be a code owner for your language, please reply below.

I've tagged people who have contributed in the past below. I just made this list by skimming through old PRs so I might have missed some people. If you've contributed in the past (writing or reviewing) and I forgot to tag you, or you notice that someone else who contributed to your language is not on the list, please let me know and I will tag them as well.

AR
@a4addel
@Sboonny

FR
@Defranos
@joachimjusth

NO
@estubmo
@josephayman

PO
@matibox
@Infiplaya
@PiotrekPKP

PT
@claudiofreitas
@minsk-dev
@Sn0wye
@victoriaquasar

RU:
@AmadeusTwi
@ronanru
@Yevhenii-Bakhmat

ZH-HANS:
@fernandoxu
@lodisy

### PR DESCRIPTION
**Pull request by juliusmarminge**
Closes #1121

## ✅ Checklist

- [ ] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [ ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

**Comment by juliusmarminge** [comment]
> 



> Hey t3-oss/translators!
> 
> This PR contains changes to your language. Please review the changes ❤️.
> 
> _estubmo _josephayman _matibox _Infiplaya _PiotrekPKP

Will add the `@` later - don't want to ping everyone unnecessarily during testing

File: .github/workflows/translations.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      - name: Write comment on the PR tagging codeowners
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          number: ${{ env.WORKFLOW_RUN_PR }}
+          message: |
+            Hey t3-oss/translators! 
+
+            This PR contains changes to your language. Please review the changes ❤️.
+
+            ${{ env.CODEOWNERS }}
```
Comment (juliusmarminge) [inline .github/workflows/translations.yml:62]: Format this message however

File: .github/workflows/translations.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+            const CODEOWNERS = {
+              ar: ["a4addel", "Sboonny"],
+              fr: ["Defranos", "joachimjusth"],
+              no: ["estubmo", "josephayman"],
+              pl: ["matibox", "Infiplaya", "PiotrekPKP"],
+              pt: ["claudiofreitas", "minsk-dev", "Sn0wye", "victoriaquasar"],
+              ru: ["AmadeusTwi", "ronanru", "Yevhenii-Bakhmat"],
+              "zh-hans": ["fernandoxu", "lodisy"],
+            };
```
Comment (juliusmarminge) [inline .github/workflows/translations.yml:41]: add codeowners here

**Review by c-ehrlich — APPROVED** [review]
I added a note about the code owners setup to TRANSLATIONS.md

Everything else looks good to me, I think the message that gets posted is fine as is. Ready to merge?

**Pull request merged**